apps/ssr-tests/just.config.js

```diff
@@ -10,4 +10,7 @@ task('mocha', () => {
   return spawn(process.execPath, [mochaCmd, 'dist/ssr-tests.js'], { stdio: 'inherit' });
 });
 
-task('build', series('webpack', 'mocha')).cached();
+// TODO: add typescript task? shouldn't have to bundle for typechecking
```

Comment:
Does webpack even do type-checking in this project's configuration? (IIRC we typically only do type-checking in webpack serve config)

Reply:
I'll check. I just assumed typechecking is something we'd want in this package, unless there's a reason not to that I don't know of. (Is ssr-test 100% auto-generated? Perhaps that's why.)

Reply:
Yeah, `ssr-tests` has no TypeScript. I'll remove the comment.